pkg/k8sutil/pod_test.go

```diff
@@ -0,0 +1,132 @@
+package k8sutil
+
+import (
+	"testing"
+
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+func TestPodUsesPVC(t *testing.T) {
+	for _, tt := range []struct {
+		name     string
+		expected bool
+		pod      corev1.Pod
+		pvc      corev1.PersistentVolumeClaim
+	}{
+		{
+			name:     "uses pvc",
```

Comment:
wdyt about the name of the tests describes what should test and the expected result?
(i.e should pass when uses pvc and has enough disk space)

Reply:
fixed.